Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1473,27 +1507,29 @@ static void addUserInstalledFontURLs(NSString *path, Vector<URL>& fontURLs)
     if (!fontNames)
         return;
 
-    Vector<URL> fontURLs;
+    if (!m_userInstalledFontURLs) {
+        m_userInstalledFontURLs = { };
```

Comment:
This looks like a no-op? if we're here then `m_userInstalledFontURLs` is nullopt and will still be nullopt after this assignment.